src/vit-testing/mainnet-tools/src/cardano_cli/fake.rs

```diff
@@ -104,72 +130,76 @@ impl Default for QueryUTxO {
                     tx_hash: "61d47e568b1502064906e977aae848c7aec9a76f97e7d11ad5d752e95c438011"
                         .to_string(),
                     tx_ix: 0,
-                    amount: 1379280,
+                    amount: 1_379_280,
                 },
                 UTxO {
                     tx_hash: "ac1d8802a4e100d90ce59fb4e4573f1c7884a65197ff39810a88eb0b07de3aa6"
                         .to_string(),
                     tx_ix: 0,
-                    amount: 30000000,
+                    amount: 30_000_000,
                 },
                 UTxO {
                     tx_hash: "69818d49963ffafe8a287ec270d05ba89493de33ddf7b5b9bcb07e97802a0f28"
                         .to_string(),
                     tx_ix: 0,
-                    amount: 5573009,
+                    amount: 5_573_009,
                 },
                 UTxO {
                     tx_hash: "fba1526c49684722199b102bffd5b4a66ea1d490605532753fa24e12af925722"
                         .to_string(),
                     tx_ix: 0,
-                    amount: 5000000,
+                    amount: 5_000_000,
                 },
             ],
         }
     }
 }
 
+/// Arbitrary protocol parameters
+#[must_use]
 pub fn protocol_parameters() -> Value {
     json!({
         "txFeePerByte": 44,
-        "minUTxOValue": 34482,
+        "minUTxOValue": 34_482,
         "decentralization": 0,
-        "utxoCostPerWord": 34482,
-        "stakePoolDeposit": 500000000,
+        "utxoCostPerWord": 34_482,
+        "stakePoolDeposit": 500_000_000,
         "poolRetireMaxEpoch": 18,
         "extraPraosEntropy": null,
         "collateralPercentage": 150,
         "stakePoolTargetNum": 500,
-        "maxBlockBodySize": 73728,
-        "minPoolCost": 340000000,
-        "maxTxSize": 16384,
+        "maxBlockBodySize": 73_728,
+        "minPoolCost": 340_000_000,
+        "maxTxSize": 16_384,
         "treasuryCut": 0.2,
         "maxBlockExecutionUnits": {
-        "memory": 50000000,
-        "steps": 4000000
+        "memory": 50_000_000,
+        "steps": 4_000_000
```

Comment:
Not sure if `rustfmt` can do this automatically, but having an extra indent here would be nice for visualizing the structure